test/runtests.jl

```diff
@@ -5,6 +5,37 @@ using POMDPModels
 using Cairo # to make Compose work
 
 @testset "Basic call checks" begin
-    @test_nowarn makegif(SimpleGridWorld())
-    @test_nowarn makegif(SimpleGridWorld(), extra_initial=true, extra_final=true)
+    try
+        makegif(SimpleGridWorld(), filename="test1.gif")
+        rm("test1.gif")
+        @test true
+    catch
+        @test false
+    end
```

Comment:
```suggestion
@test mktempdir() do dir
    makegif(SimpleGridWorld(), filename=joinpath(dir, "test1.gif"))
    return true
end
```

Reply:
I like the idea of using `mktempdir`. As I had it structured it would also "check" if the file existed because `rm` throws an error if the file doesn't exist. 

In a hybrid of the two methods, I modified the tests in the last commit. This version uses `mktempdir` to ensure cleanup after running the tests but also explicitly checks for the file existing.